fastapi_standalone_docs/scripts.py

```diff
@@ -0,0 +1,59 @@
+from pathlib import Path
+from typing import List
+from urllib.request import urlretrieve
+
+import fastapi_standalone_docs
+
+FASTAPI_FILES = [
+    "https://fastapi.tiangolo.com/img/favicon.png",
+    "https://raw.githubusercontent.com/tiangolo/fastapi/master/LICENSE",
+]
+
+SWAGGER_FILES = [
+    "https://cdn.jsdelivr.net/npm/swagger-ui-dist@4/LICENSE",
+    "https://cdn.jsdelivr.net/npm/swagger-ui-dist@4/swagger-ui.css",
+    "https://cdn.jsdelivr.net/npm/swagger-ui-dist@4/swagger-ui-bundle.js",
+]
+
+REDOC_FILES = [
+    "https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js.LICENSE.txt",
+    "https://cdn.jsdelivr.net/npm/redoc@next/bundles/redoc.standalone.js",
```

Comment:
Should these really point to `@next`?

Reply:
That's a good question. I actually didn't look that close, just copy&pasted what was used in the latest FastAPI release. This is the code in question: https://github.com/tiangolo/fastapi/blob/0.97.0/fastapi/openapi/docs.py#L79. It seems like that code was committed some 4 years ago, so way before the ReDoc 2.0.0 release was made (it was released 9 months ago). I thus believe using the 2.0.0 version would be preferable. I'll test it to confirm it seems to work, switch to it and likely make a PR to FastAPI as well to lock down the version to the stable version there.

Reply:
Seems like the `next` tag at the current time pointed to a pre-release version of 2.0.0, so updated to 2.0.0 and tested it seemed to work OK. Also made a PR to FastAPI to update it there to 2.0.0. I have a small hunch that tiangolo might prefer to use the `latest` tag, but we'll see.